webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingApplications.vue

```diff
@@ -60,13 +60,27 @@ export default {
 
       this.$spaceService.getSpaceApplications(this.spaceId)
         .then(applications => {
-          this.applications = applications;
+          this.applications = applications.map(app => ({
+            applicationName: app.applicationName,
+            contentId: app.contentId,
+            description: this.$t(`SpaceSettings.application.${/\s/.test(app.displayName) ? app.displayName.replace(/ /g,'.').toLowerCase() : app.displayName.toLowerCase()}.description`),
+            displayName: this.$t(`SpaceSettings.application.${/\s/.test(app.displayName) ? app.displayName.replace(/ /g,'.').toLowerCase() : app.displayName.toLowerCase()}.title`),
+            id: app.id(),
```

Comment:
A regression is made by this, the `app.id` is an attribute not a function.
(
Try to add an application in a space or delete it => 
```
Uncaught (in promise) TypeError: app.id is not a function
    at eval (SpaceSettingApplications.vue?7190:68)
    at Array.map (<anonymous>)
    at eval (SpaceSettingApplications.vue?7190:63)
```)

Reply:
Instead of altering retrieved data from backend, you should have added a `computed` attribute and displayed in this sub Vue component :
https://github.com/Meeds-io/social/blob/1e9ac24366785aa625da87a86ff80edd4f792899/webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingApplicationCard.vue#L13-L22

Reply:
> A regression is made by this, the `app.id` is an attribute not a function.
> (
> Try to add an application in a space or delete it =>
> 
> ```
> Uncaught (in promise) TypeError: app.id is not a function
>     at eval (SpaceSettingApplications.vue?7190:68)
>     at Array.map (<anonymous>)
>     at eval (SpaceSettingApplications.vue?7190:63)
> ```)
> ```

I didn't notice it, this is probably due to the auto-completion.